app/models/pillar.rb

```diff
@@ -36,4 +36,23 @@ class Pillar < ApplicationRecord
   validates :value, presence: true
 
   scope :global, -> { where minion_id: nil }
+
+  class << self
+    def value(pillar:)
+      pillar = Pillar.find_by pillar: all_pillars[pillar]
```

Comment:
I suggest we don't overwrite the original argument because it was a string and now it is a Pillar object. It makes it harder to mentally run the code because have to keep track of what the variable is at each point.